### DISCUSSION TITLE
Slip flow

### DISCUSSION THREAD

**Discussion by MingyueTianya1** [PMEAL/OpenPNM#2806] [Q&A]
Dear Professor，

I was wondering if you have an example of slip flow？Or where can I learn about slip flow？

Thanks in advance for your guidance.

**Comment by jgostick** [comment]
We do not have an example, but the idea is fairly simple.  It can be modeled by imagining that the parabolic velocity profile extends into the wall a bit like below:

![image](https://github.com/PMEAL/OpenPNM/assets/5092049/606e28d1-95c4-4f2b-88d6-e3823a997034)

**Reply by MingyueTianya1** [reply]
Thank you very much for your reply to the problem, which benefit me a lot. And if I want to add a wetting films in the corners, where can I learn that?
